examples/acceptance_tests/071/plutus.json

```diff
@@ -19,8 +19,8 @@
+      "hash": "b66a8c385bf85828be1b5811b535fd381ec4197b822248356e29b371"
```

Comment:
What's happening here 🤔 ?

Reply:
Now the way to make a small script would be to use data as the type